lib/utilities/validators.dart

```diff
@@ -0,0 +1,3 @@
+final oneToFifteenChars = (String value) => value.isEmpty
```

Comment:
love that you put this into a utilities folder, makes me want to refactore the create user account screen

Reply:
Thanks!  I liked this pattern too - top level functions in a utilities file and then import with a prefix... you don't even need a class but avoid namespace collisions